### DISCUSSION TITLE
Escape sequence

### DISCUSSION THREAD

**Discussion by Omcsesz** [community/community#18748] [Other Feature Feedback, Questions, & Ideas]
Hello, fellow users!

Is there a way to escape the "===================================" sequence in text mode? I want to write a console log message which contains this character sequence, but it is not displayed correctly...

Thanks in advance.

**Comment by Zerotask** [comment]


**Reply by Omcsesz** [reply]
No, I wanted to write the following:

``` text
.... some generic text ...
=======================================
.... some generic text ...
```

**Reply by Zerotask** [reply]
If I open the browser console and paste this, I see it properly.

```js
console.log('.... some generic text ...');
console.log('=======================================');
console.log('.... some generic text ...');
```

**Reply by Omcsesz** [reply]
I want to write this in the Github on-site editor.

**Reply by Zerotask** [reply]
I created a new repo, added a test code and clicked on edit file
https://github.com/Zerotask/Escape-sequence-test/blob/main/test.js
![image](https://user-images.githubusercontent.com/20150243/174099576-6e32aeed-dbf6-48ad-a7c2-503a7075e5cb.png)

I still don't get it what exactly your problem is. Can you please describe it more detailed?

**Reply by Zerotask** [reply]
Here's also an issue example
https://github.com/Zerotask/Escape-sequence-test/issues/1
It shows it as expected

**Reply by Omcsesz** [reply]
:D :D No this is not what I'm talking about.
I talk about the following file:
https://github.com/zkoss/zk-mvvm-book/edit/7.0/configuration/README.md

If you look in it, there is a section:

` ` ` text
.... 

In that section there is a console message with a line of =s, which is not rendered properly.

**Reply by Zerotask** [reply]
> :D :D No this is not what I'm talking about.

Then describe exactly at the beginning what the problem is and include references instead of making fun of someone who tries to help you but don't understand you.

I can't click the provided link since I'm not a contributor. Also I don't know what you mean with:
![image](https://user-images.githubusercontent.com/20150243/174110369-e1abc1f8-be98-40f7-81cb-531cb28696c4.png)

If I open https://github.com/zkoss/zk-mvvm-book/tree/7.0/configuration then you probably mean 
![image](https://user-images.githubusercontent.com/20150243/174111692-1c855c28-e06f-4ee8-8c23-bb59228d5d03.png)

which is a console output. And what exactly is wrong there? Just make a screenshot and highlight it, make a "current" and "expected" etc so people can actually help you.

> In that section there is a console message with a line of =s, which is not rendered properly.

there is no `=s` on that page (ctrl + f)

**Reply by Omcsesz** [reply]
> I can't click the provided link since I'm not a contributor. Also I don't know what you mean with: 

I'm sorry, I did not know you could not click on it.
This is the issue I mentioned: 
![image](https://user-images.githubusercontent.com/1839739/174113169-86df8e87-a0bc-4f38-a83d-074c8daaa617.png)


> If I open https://github.com/zkoss/zk-mvvm-book/tree/7.0/configuration then you probably mean ![image](https://user-images.githubusercontent.com/20150243/174111692-1c855c28-e06f-4ee8-8c23-bb59228d5d03.png)
> 
> which is a console output. And what exactly is wrong there? Just make a screenshot and highlight it, make a "current" and "expected" etc so people can actually help you.

The problem is that it is rendered like this:
![image](https://user-images.githubusercontent.com/1839739/174114219-22a6c05b-89c0-4ad5-b015-8c77664b8748.png)


> 
> > In that section there is a console message with a line of =s, which is not rendered properly.
> 
> there is no `=s` on that page (ctrl + f)
Yes, that is correct, but there is a line of equal signs (I shortened it like =s, sorry for the ambiguity)
